<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiDWidget</class>
 <widget class="QWidget" name="MultiDWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>992</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>472</width>
        <height>924</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="save_groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="title">
          <string>Save track</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Directory:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="dir_lineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="browse_save_Button">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>File Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="fname_lineEdit">
              <property name="text">
               <string>Experiment</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <property name="title">
          <string>Peak tracker</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="9" column="3">
           <widget class="QCheckBox" name="show_on_napari_checkBox">
            <property name="text">
             <string>show in napari</string>
            </property>
           </widget>
          </item>
          <item row="10" column="3">
           <widget class="QCheckBox" name="plot_trace_checkBox">
            <property name="text">
             <string>Plot trace</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>display memory (s)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3">
           <widget class="QComboBox" name="layer_comboBox"/>
          </item>
          <item row="9" column="2">
           <widget class="QCheckBox" name="roi_checkBox">
            <property name="text">
             <string>show ROI</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_18">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>layer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>model</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0" colspan="4">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="title">
             <string>Configure ROIs</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="0" rowspan="3">
              <widget class="QTableWidget" name="fit_roi_tableWidget">
               <column>
                <property name="text">
                 <string>cy</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>cx</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>size</string>
                </property>
               </column>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="add_pushButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="clear_pushButton">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="remove_pushButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="mouse_select_pushButton">
               <property name="text">
                <string>Select with mouse</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QDoubleSpinBox" name="memory_time_doubleSpinBox">
            <property name="maximum">
             <double>9999999999999999455752309870428160.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="model_comboBox"/>
          </item>
          <item row="9" column="1">
           <widget class="QCheckBox" name="compare_checkBox">
            <property name="text">
             <string>compare to center</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="3">
           <widget class="QPushButton" name="update_pushButton">
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QDoubleSpinBox" name="update_rate_doubleSpinBox">
            <property name="maximum">
             <double>9999999999999999635896294965248.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>update rate (ms)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="run_btn_layout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="show_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>SHOW TRACE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_Button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_Button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>save_groupBox</tabstop>
  <tabstop>dir_lineEdit</tabstop>
  <tabstop>browse_save_Button</tabstop>
  <tabstop>fname_lineEdit</tabstop>
  <tabstop>model_comboBox</tabstop>
  <tabstop>layer_comboBox</tabstop>
  <tabstop>update_pushButton</tabstop>
  <tabstop>fit_roi_tableWidget</tabstop>
  <tabstop>add_pushButton</tabstop>
  <tabstop>remove_pushButton</tabstop>
  <tabstop>clear_pushButton</tabstop>
  <tabstop>show_Button</tabstop>
  <tabstop>run_Button</tabstop>
  <tabstop>pause_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
